pages/workspace/tracking-plan/properties.mdx

```diff
@@ -22,9 +22,23 @@ relevant for that specific event, when you analyze your data. You might be want
 
 Event properties are sent with the event in the <Link passHref href="/workspace/tracking-plan/events#a-namelog-eventa-log-event"><a>event call</a></Link>.
 
+### <a name="group-properties"></a> Group properties
+
+Group properties describe the state of a group you defined previously. Read more about Groups <Link passHref href="/data-design/groups"><a>here</a></Link>.
```

Comment:
```suggestion
Group properties describe the state of a group you defined previously.
```

Reply:
I feel like we need to introduce groups for better understanding of the group properties concept

Reply:
Totally agree. Was suggesting to move it into a highlights section here https://github.com/avohq/docs/pull/209#discussion_r681735359 

whatcha think?

Reply:
Moved!